### PR TITLE
[Room list] Use the items from Element instead of the duplicated Tchap ones

### PR DESCRIPTION
**Pull request by Florian14**
#386 

image

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItem.kt [hunk 1]
```diff
@@ -109,20 +127,69 @@ abstract class RoomSummaryItem : VectorEpoxyModel<RoomSummaryItem.Holder>() {
...
         val typingView by bind<TextView>(R.id.roomTypingView)
         val draftView by bind<ImageView>(R.id.roomDraftBadge)
         val lastEventTimeView by bind<TextView>(R.id.roomLastEventTimeView)
         val avatarCheckedImageView by bind<ImageView>(R.id.roomAvatarCheckedImageView)
         val avatarImageView by bind<ImageView>(R.id.roomAvatarImageView)
-        val roomAvatarDecorationImageView by bind<ShieldImageView>(R.id.roomAvatarDecorationImageView)
-        val roomAvatarPublicDecorationImageView by bind<ImageView>(R.id.roomAvatarPublicDecorationImageView)
-        val roomAvatarFailSendingImageView by bind<ImageView>(R.id.roomAvatarFailSendingImageView)
+
+        //        val roomAvatarDecorationImageView by bind<ShieldImageView>(R.id.roomAvatarDecorationImageView)
```
Comment (ClaireGizard) [inline vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItem.kt:183]: reformat this comment

File: vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItem.kt [hunk 2]
```diff
@@ -109,20 +127,69 @@ abstract class RoomSummaryItem : VectorEpoxyModel<RoomSummaryItem.Holder>() {
...
+            resource?.let { setImageDrawable(ContextCompat.getDrawable(holder.view.context, it)) }
+            visibility = if (resource != null) View.VISIBLE else View.INVISIBLE
+        }
+    }
+
     class Holder : VectorEpoxyHolder() {
         val titleView by bind<TextView>(R.id.roomNameView)
-        val unreadCounterBadgeView by bind<UnreadCounterBadgeView>(R.id.roomUnreadCounterBadgeView)
+
+        //        val unreadCounterBadgeView by bind<UnreadCounterBadgeView>(R.id.roomUnreadCounterBadgeView)
```
Comment (ClaireGizard) [inline vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItem.kt:174]: reformat the comment

File: vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItemFactory.kt [hunk 3]
```diff
@@ -98,7 +110,7 @@ class RoomSummaryItemFactory @Inject constructor(private val displayableEventFor
                 .changeMembershipState(changeMembershipState)
                 .acceptListener { listener?.onAcceptRoomInvitation(roomSummary) }
                 .rejectListener { listener?.onRejectRoomInvitation(roomSummary) }
-                .listener { listener?.onRoomClicked(roomSummary) }
+//                .listener { listener?.onRoomClicked(roomSummary) }
```
Comment (ClaireGizard) [inline vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItemFactory.kt:113]: maybe a comment here ?

File: vector/src/main/res/layout/item_tchap_room.xml [hunk 4]
```diff
@@ -8,7 +8,8 @@
     android:background="?android:colorBackground"
     android:clickable="true"
     android:focusable="true"
-    android:foreground="?attr/selectableItemBackground">
+    android:foreground="?attr/selectableItemBackground"
+    tools:viewBindingIgnore="true">
```
Comment (ClaireGizard) [inline vector/src/main/res/layout/item_tchap_room.xml:12]: why did you add this line

File: vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItemFactory.kt (hunk 3 above)
Comment (ClaireGizard) [reply vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItemFactory.kt:113]: it was added by Element ?

**Review by Claire1817 — COMMENTED** [review]
Will fix: https://github.com/tchapgouv/tchap-android-v2/issues/419

File: vector/src/main/res/layout/item_tchap_room.xml (hunk 4 above)
Comment (Florian14) [reply vector/src/main/res/layout/item_tchap_room.xml:12]: it's a change from element

File: vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItemFactory.kt (hunk 3 above)
Comment (Florian14) [reply vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItemFactory.kt:113]: Nop, specific Tchap, added a comment

File: vector/src/main/res/layout/item_tchap_room.xml [hunk 5]
```diff
@@ -93,166 +75,134 @@
...
-        app:layout_constraintBottom_toBottomOf="@id/roomAvatarContainer"
+        android:layout_marginEnd="8dp"
+        android:layout_marginBottom="8dp"
+        android:textColor="?vctr_content_secondary"
+        app:layout_constraintBottom_toTopOf="@id/roomBottomBarrier"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/roomFailSendingImageView"
         tools:text="@tools:sample/date/hhmm" />
 
     <TextView
```
[the remark this answers is not in the document]
Comment (Florian14) [reply vector/src/main/res/layout/item_tchap_room.xml:177]: Not in Element-1.3.12 :-)

File: vector/src/main/res/layout/item_tchap_room.xml [hunk 6]
```diff
@@ -93,166 +75,134 @@
...
         android:textColor="?vctr_content_secondary"
-        app:layout_constraintBottom_toBottomOf="@id/roomAvatarContainer"
-        app:layout_constraintEnd_toStartOf="@id/roomLastEventTimeView"
-        app:layout_constraintStart_toStartOf="@id/roomNameContainer"
+        app:layout_constraintEnd_toStartOf="@id/tchapRightBarrier"
+        app:layout_constraintStart_toStartOf="@id/roomNameView"
+        app:layout_constraintTop_toBottomOf="@id/roomNameView"
         tools:text="@sample/messages.json/data/message" />
 
     <TextView
```
[the remark this answers is not in the document]
Comment (Florian14) [reply vector/src/main/res/layout/item_tchap_room.xml:192]: same here

**Review by Claire1817 — APPROVED** [review]
